### PR TITLE
Release 2.3.0

### LINKED ISSUE #1612: Release 2.3.0

### PR DESCRIPTION
**Pull request by aristov**
Closes #1612.
cc @narqo @veged @dfilatov

### CODE REVIEW

**Comment by aristov** [comment]
cc @tadatuta @innabelaya

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 2.3.0
+
+### Major changes
+* `bem-core` dependency was updated to [2.7.0](https://github.com/bem/bem-core/blob/v2.7.0/CHANGELOG.ru.md#270).
```
Comment (tadatuta) [inline CHANGELOG.md]: https://en.bem.info/libs/bem-core/v2.7.0/changelog/#270

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+* BH 4.x support added ([#1587](https://github.com/bem/bem-components/issues/1587)).
+* BEM-XJST 2.x support added ([#1495](https://github.com/bem/bem-components/pull/1495)).
+
+### Bug fixes
+* An issue in `checkbox` was fixed. The block didn't work in MSIE 11/Edge ([#1590](https://github.com/bem/bem-components/issues/1590)).
+* An issue in `attach` was fixed. The block didn't work in MSIE 11/Edge ([#1596](https://github.com/bem/bem-components/issues/1596)).
+* Theme-specific styles of `input`, `textarea` and `select` of the `_width_available` modifier was moved back from `common` level to `design` ([#1548](https://github.com/bem/bem-components/issues/1548)).
```
Comment (tadatuta) [inline CHANGELOG.md]: This is kinda breaking change and there are people who use it as is was in `2.6.0`, let's move it to major section or highlight somehow.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,26 @@
...
+* An issue in `button` was fixed. An ability to use HTML tags other than `button` was added ([#1566](https://github.com/bem/bem-components/issues/1566)).
```
Comment (tadatuta) [inline CHANGELOG.md]: Let's get rid of the first sentense here and in the next item?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
+* An issue in `textarea` was fixed. The block didn't resolve it's own dependencies properly ([#1565](https://github.com/bem/bem-components/issues/1565)).
+
+### Other changes
+* The English version of library description was updated ([#1552](https://github.com/bem/bem-components/pull/1552)).
+* New "Understanding of the library principles" section was added to the russian documentation ([#1613](https://github.com/bem/bem-components/pull/1613)).
```
Comment (tadatuta) [inline CHANGELOG.md]: "Understanding of the library principles" -> “Understanding of the library principles” (typography)

File: CHANGELOG.ru.md [hunk 5]
```diff
@@ -1,5 +1,26 @@
 # История изменений
 
+## 2.3.0
+
+### Крупные изменения
+* Библиотека `bem-core` была обновлена до версии [2.7.0](https://github.com/bem/bem-core/blob/v2.7.0/CHANGELOG.ru.md#270).
```
Comment (tadatuta) [inline CHANGELOG.ru.md]: https://ru.bem.info/libs/bem-core/v2.7.0/changelog/#270

File: CHANGELOG.ru.md [hunk 6]
```diff
@@ -1,5 +1,26 @@
...
+### В релиз вошли следующие исправления ошибок
+* В `checkbox` иправлена ошибка в MSIE 11/Edge ([#1590](https://github.com/bem/bem-components/issues/1590)).
+* В `attach` иправлена ошибка в MSIE 11/Edge ([#1596](https://github.com/bem/bem-components/issues/1596)).
+* В `input`, `textarea` и `select` специфичные для темы стили модификатора `_width_available` вынесены из `common`-уровня обратно на `design`-уровень переопределения ([#1548](https://github.com/bem/bem-components/issues/1548)).
+* В `button` исправлена поддержка нестандартных HTML-тегов для собственных реализаций кнопки ([#1566](https://github.com/bem/bem-components/issues/1566)).
+* В `textarea` исправлена ошибка, из-за которой блок неправильно обрабатывал собственные зависимости ([#1565](https://github.com/bem/bem-components/issues/1565)).
+
+### Также в релиз вошли следующие изменения
+* Обновлена английская версия описания библиотеки ([#1552](https://github.com/bem/bem-components/pull/1552)).
+* В русскую документацию добавлен раздел "Понимание принципов библиотеки" и внесены другие мелкие исправления ([#1613](https://github.com/bem/bem-components/pull/1613)).
```
Comment (tadatuta) [inline CHANGELOG.ru.md]: «»

**Comment by voischev** [comment]
:fire:

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,26 @@
...
+* `bem-core` dependency was updated to [2.7.0](https://github.com/bem/bem-core/blob/v2.7.0/CHANGELOG.ru.md#270).
+* BH 4.x support added ([#1587](https://github.com/bem/bem-components/issues/1587)).
+* BEM-XJST 2.x support added ([#1495](https://github.com/bem/bem-components/pull/1495)).
+
+### Bug fixes
+* An issue in `checkbox` was fixed. The block didn't work in MSIE 11/Edge ([#1590](https://github.com/bem/bem-components/issues/1590)).
+* An issue in `attach` was fixed. The block didn't work in MSIE 11/Edge ([#1596](https://github.com/bem/bem-components/issues/1596)).
+* Theme-specific styles of `input`, `textarea` and `select` of the `_width_available` modifier was moved back from `common` level to `design` ([#1548](https://github.com/bem/bem-components/issues/1548)).
+* An issue in `button` was fixed. An ability to use HTML tags other than `button` was added ([#1566](https://github.com/bem/bem-components/issues/1566)).
+* An issue in `textarea` was fixed. The block didn't resolve it's own dependencies properly ([#1565](https://github.com/bem/bem-components/issues/1565)).
```
Comment (tadatuta) [inline CHANGELOG.md:15]: This was not actually a bug but we decided it's too complicated to set up build config properly. So it's more like improvement than a bug fix, isn't it?

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 2.3.0
```
Comment (tadatuta) [inline CHANGELOG.md:3]: new pr was merged: https://github.com/bem/bem-components/issues/1610

File: CHANGELOG.md (hunk 7 above)
Comment (narqo) [inline CHANGELOG.md:15]: It was a bug, as `texarea` hadn't been working without this changes.

File: CHANGELOG.md (hunk 8 above)
Comment (sipayRT) [inline CHANGELOG.md:3]: I want to merge one more RP before release. It's about bug with selected text in FF38-39.

Comment (sipayRT) [inline CHANGELOG.md:3]: Ready! https://github.com/bem/bem-components/issues/1608

File: CHANGELOG.md (hunk 1 above)
Comment (aristov) [inline CHANGELOG.md]: Previously [we gave the GitHub link](https://github.com/bem/bem-components/blob/v2/CHANGELOG.md#major-changes). Are you sure that we should deviate from this now?

File: CHANGELOG.ru.md (hunk 5 above)
Comment (aristov) [inline CHANGELOG.ru.md]: Previously [we gave the GitHub link](https://github.com/bem/bem-components/blob/v2/CHANGELOG.md#major-changes). Are you sure that we should deviate from this now?

Comment (tadatuta) [inline CHANGELOG.ru.md]: I think bem.info is much better way to read library documentation so why shouldn't we force users to it?

File: CHANGELOG.md (hunk 8 above)
Comment (aristov) [inline CHANGELOG.md:3]: > new pr was merged: #1610

Added.

> Ready! #1608

Added.

File: CHANGELOG.ru.md (hunk 5 above)
Comment (aristov) [inline CHANGELOG.ru.md]: Ok, fixed. English version fixed too.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,28 @@
...
+
+### Major changes
+* `bem-core` dependency was updated to [2.7.0](https://en.bem.info/libs/bem-core/v2.7.0/changelog/#270).
+* BH 4.x support added ([#1587](https://github.com/bem/bem-components/issues/1587)).
+* BEM-XJST 2.x support added ([#1495](https://github.com/bem/bem-components/pull/1495)).
+* Theme-specific styles of `input`, `textarea` and `select` of the `_width_available` modifier was moved back from `common` level to `design` ([#1548](https://github.com/bem/bem-components/issues/1548)).
```
Comment (innabelaya) [inline CHANGELOG.md]: может, Theme-specific styles of the `_width_available` modifier of `input`, `textarea` and `select` blocks was .....

Comment (tadatuta) [inline CHANGELOG.md]: если так, то blocks ~~was~~ were

Comment (aristov) [inline CHANGELOG.md]: Поправил.

**Pull request merged**